### DISCUSSION TITLE
Swell Transaction Fee and Returns

### DISCUSSION THREAD

**Discussion by wloodar** [swellstores/community#149] [Help] [answered]
Hi!
Does swell charge a transaction fee for refunds, so when customer decides to return their order, will merchant still be charged for that order?

Recently I spent some time discovering possibilities coming with swell and it's no-brainer in my case - exciting and superb solution!

Only this one thing with fee and returns is not 100% clear to me, so grateful for help!

**Comment by swellmike** [comment, marked as answer]
hi @wloodar, I assume you are referring to revenue fees associated with Swell plans.  Revenue fees are tied to net sales, so canceled orders are excluded.

**Reply by wloodar** [reply]
Great to hear that, thank you!
